src/Tabs.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TabsListTab
+ *
+ * Direct child of `TabsList`.
+ * Renders individual tab control.
+ * Accepts a `selected` prop.
+ */
+export const TabsListTab = React.createClass({
```

Comment:
It's too bad we can't use the more compact definitions in the body of the class... maybe there's a babel setting we can flip later on. Thanks for taking a look at this!